src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
How will you treat things in a branch and non-filterable? A property like branch_id is what you are suggesting? That makes it look like more complicated.